.github/workflows/CI-e4s.yml

```diff
@@ -0,0 +1,149 @@
+name: CI-e4s
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+
+jobs:
+  HIP:
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        cxx: ['hipcc']
+        cmake_build_type: ['Debug', 'Release']
+        kokkos_ver: ['3.6.01']
+        arborx: ['OFF']
+        heffte: ['OFF' ]
+        hypre: ['OFF']
+        liball: ['OFF']
+        silo: ['OFF']
+        hdf5: ['OFF']
+    runs-on: ubuntu-20.04
+    container: docker.io/ecpe4s/e4s-base-rocm:23.05
+    steps:
+      - name: Checkout GTest
+        uses: actions/checkout@v3
+        with:
+          repository: google/googletest
+          ref: release-1.11.0
+          path: gtest
+      - name: Build gtest
+        working-directory: gtest
+        run: |
+          cmake -B build \
+            -DCMAKE_INSTALL_PREFIX=$HOME/gtest \
+            -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}
+          cmake --build build --parallel 2
+          cmake --install build
+      - name: Checkout kokkos
+        uses: actions/checkout@v3
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ matrix.kokkos_ver }}
+          path: kokkos
+      - name: Build kokkos
+        working-directory: kokkos
+        run: |
+          cmake -B build \
+            -DCMAKE_INSTALL_PREFIX=$HOME/kokkos \
+            -DCMAKE_CXX_COMPILER=${{ matrix.cxx }} \
+            -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }} \
+            -DKokkos_ENABLE_HIP=ON \
+            -DKokkos_ARCH_VEGA908=ON
+          cmake --build build --parallel 2
+          cmake --install build
+      - name: Checkout arborx
+        if: ${{ matrix.arborx == 'ArborX' }}
+        uses: actions/checkout@v3
+        with:
+          repository: arborx/ArborX
+          ref: v1.2
+          path: arborx
+      - name: Build arborx
+        if: ${{ matrix.arborx == 'ArborX' }}
+        working-directory: arborx
+        run: |
+          cmake -B build \
+            -DCMAKE_PREFIX_PATH=${HOME}/kokkos \
+            -DCMAKE_INSTALL_PREFIX=$HOME/arborx \
+            -DCMAKE_CXX_COMPILER=${{ matrix.cxx }} \
+            -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}
+          cmake --build build --parallel 2
+          cmake --install build
+      - name: Checkout heffte
+        if: ${{ matrix.heffte != 'OFF' }}
+        # actions/checkout doesn't work for external repos yet (actions/checkout#447)
+        run: |
+          git clone --depth 1 --branch v2.1.0 https://bitbucket.org/icl/heffte.git heffte
+      - name: Build heffte
+        if: ${{ matrix.heffte != 'OFF' }}
+        working-directory: heffte
+        run: |
+          cmake -B build \
+            -DCMAKE_CXX_STANDARD="11" \
```

Comment:
Any good reason to enforce C++11 for heffte?

Reply:
No - removed